pymeasure/instruments/agilent/agilentB1500.py

```diff
@@ -427,8 +423,8 @@ def format_channel_check_status(self, status_string, channel_string):
     class _data_formatting_FMT1(_data_formatting_generic):
         """ Data formatting for FMT1 format
         """
-        def __init__(self, smu_names={}):
-            super().__init__("FMT1", smu_names)
+        def __init__(self, smu_names={}, output_format_string="FMT1"):
```

Comment:
If nothing opposes this, maybe switch `smu_names`and `output_format_string` parameter order, so that it matches the superclass's?

Reply:
good point, I switched the arguments of the super class and dropped the defaults there. For the derived classes it is desirable to have `smu_names` as first argument, since `output_format_string` is only used when deriving additional classes and can be omitted in most cases.